src/category_theory/groupoid.lean

```diff
@@ -81,6 +84,10 @@ noncomputable
 def groupoid.of_is_iso (all_is_iso : ∀ {X Y : C} (f : X ⟶ Y), is_iso f) : groupoid.{v} C :=
 { inv := λ X Y f, inv f }
 
+/-- A category with a unique morphism between any two objects is a groupoid -/
+def groupoid.of_all_unique (all_unique : ∀ {X Y : C}, unique (X ⟶ Y)) :
+  groupoid.{v} C := { inv := λ X Y f, all_unique.default }
```

Comment:
```suggestion
  groupoid.{v} C := 
{ inv := λ X Y f, all_unique.default }
```
Not a hard and fast rule (I think?) but we tend to put a newline after the `:=` unless it's very very short afterwards (e.g. `rfl`).